src/main/resources/com/puppycrawl/tools/checkstyle/messages.properties

```diff
@@ -1,12 +1,16 @@
+#
+# for Checker.java
+#
```

Comment:
This comment leads me to believe that this entire file is for Checker messages, or at least all messages up to the next comment like this; the other comments are equally unhelpful and misleading IMO. We don't need to do this, please remove these. The key should tell us everything we need to know (it does).

Reply:
Done..
Rebased on master